scipy/stats/tests/test_stats.py

```diff
@@ -1162,6 +1171,19 @@ def wkq(x, y, rank, weigher, add):
             np.random.shuffle(rank)
 
 
+class TestSpearmanrWarnings(object):
```

Comment:
do we need a new class here? you could also add `def test_warning` to the class above.

Reply:
yeah fair point-changed.

Reply:
did you push that change? I still see the class

Reply:
not sure why they didn't get pushed in the last changeset, I added them in latest push.